modules/calypsoify/_studio-wpcom.scss

```diff
@@ -0,0 +1,172 @@
+// v2.2.0
```

Comment:
As a follow up I think we can import these from color studio directly

`@import '~@automattic/color-studio/path_to_the_file';`